src/dualize.jl

```diff
@@ -50,4 +46,30 @@ function dualize(primal_model::MOI.ModelLike, dual_problem::DualProblem{T}, dual
                                   primal_objective, con_types)
 
     return dual_problem
-end
\ No newline at end of file
+end
+
+# JuMP dualize
+function dualize(model::JuMP.Model; factory::Union{OptimizerFactory, Nothing} = nothing, dual_names::DualNames = DualNames("", ""))
```

Comment:
It's probably more consistent to use a positional argument for `factory`. It also reads better with the `with_optimizer`: e.g. "Dualize model with optimizer GLPK"

Reply:
what does it mean to use a positional argument?

Reply:
like this? 
```julia
function dualize(model::JuMP.Model, factory::OptimizerFactory, dual_names::DualNames = DualNames("", ""))
```

Reply:
Yes, positional is when it's not a keyword argument

Reply:
Got it, I made two functions, one that receives a `factory` and another one that does not